SurfaceTopography/IO/Reader.py

```diff
@@ -300,6 +300,14 @@ class ReaderBase(metaclass=abc.ABCMeta):
     file (from filename or stream object), inspect its metadata and then
     request to load a topography from it. Metadata is typically extracted
     without reading the full file.
+
+    Readers should adhere to the following design rules:
+    1. Opening a file should be fast and therefore not read the whole data.
+       The data is read only when requesting it via the `topography` method.
+    2. Data corruption must be detected when opening the file. The
+       `topography` method must not fail because of file corruption issues.
+    These rules are important to allow smooth operation of the readers in
+    the web application `TopoBank`.
```

Comment:
When the user uploads the raw data, selects the channel and fills in the metadata, is the topography loaded once ? Or is it only done at the first analysis ? 

It would be good to load it once during the upload procedure for testing, so the user directly gets the corresponding error.

Reply:
It is never fully read in Topobank during this process, but it is opened in TopoBank (i.e. the reader is constructed). This is why I want all errors to occur during construction of the reader.